e2e/smoke-tests-suite/panels_smokescreen.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { e2e } from '@grafana/e2e';
+
+// TODO: Better way to generate this list (ideally including all panels >=beta state) (list is case sensitive)
+const PANELS_TO_CHECK = [
```

Comment:
rather than a hardcoded list... this could get the values from either:

```
window.grafanaBootData.settings.panels
```

of from `/api/frontend/settings` and the panel list there.

Reply:
It seems that in the test context that the window.grafanaBootData is not being initialized. For getting panel list from the api do you mean making the api call inside the test to retrieve the list of panels to loop through?